src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
 body {
+  display: flex;
   margin: 0;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+$day-size: 100px;
+$border-size: 1px;
+$padding: 10px;
+$hover-color: #ffbfcb;
```

Comment:
The variable `$hover-color` is defined but make sure it contrasts well with the `$day-color` to ensure good visibility when hovering over a day.